analytics_data_api/v0/views/learners.py

```diff
@@ -36,24 +36,29 @@ class LearnerView(CourseViewMixin, generics.RetrieveAPIView):
 
     **Response Values**
 
-        Returns the learner metadata and engagement data:
+        Returns metadata and engagement data for the learner.
 
             * username: User's username.
-            * enrollment_mode: Enrollment mode (e.g. "honor).
+            * enrollment_mode: Enrollment mode (for example, "verified").
             * name: User's full name.
-            * email: User's email.
-            * segments: Classification for this course based on engagement, (e.g. "has_potential").
+            * email: User's email address.
+            * segments: Classification, based on engagement, of this learner's
+              work in this course (for example, "highly_engaged" or
+              "struggling").
             * engagements: Summary of engagement events for a time span.
-                * videos_viewed: Number of times a video was played.
-                * problems_completed: Unique number of problems completed.
-                * problems_attempted: Unique number of problems attempted.
-                * discussions_contributed: Number of discussions (e.g. forum posts).
+                * videos_viewed: Number of times any course video was played.
+                * problems_completed: Number of unique problems the learner
+                  answered correctly.
+                * problems_attempted: Number of unique problems attempted.
+                  This is a count of the different problems the learner tried.
+                * discussions_contributed: Number of posts, responses, or
+                  comments the learner contributed to course discussions.
```

Comment:
I know we're not reviewing the API here, but this seems to be the reverse of the previous field. I would expect that this would be the total number of unique discussion topics in which the learner was the poster, responder, or commenter. The wording here is fine, I just wanted to make sure the meaning is correct.